templates/taxi/car_detail.html

```diff
@@ -3,20 +3,38 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <p><span style="font-weight: bold">Manufacturer:</span> ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})
+  </p>
+  <form>
+    {% csrf_token %}
+
+    <h1>
+      Drivers
+      {% if user in car.drivers.all %}
```

Comment:
The expression `car.drivers.all` should be `car.drivers.all()` to properly call the method and retrieve the queryset of drivers.